File: Projekt1_cpp/Projekt1_cpp/Wilk.cpp
#include "Wilk.h"

Wilk::Wilk(pair<int, int>pole, Swiat* swiat)
	:Zwierze(9, 5, 0, pole, swiat) {}

Organizm* Wilk::potomek(pair<int, int>pole) {
	return new Wilk(pole, swiat);
}

void Wilk::rysowanie() {
	cout << 'w';
}

string Wilk::getNazwa() {
	return "wilk";
}

File: Projekt2_java/src/projekt2/zwierzeta/Wilk.java
package projekt2.zwierzeta;

import projekt2.*;

import java.awt.*;

public class Wilk extends Zwierze {
    public Wilk(Pair pole, Swiat swiat) {
        super(9, 5, 0, pole, swiat);
    }

    @Override
    public Organizm potomek(Pair pole) {
        return new Wilk(pole, swiat);
    }

    @Override
    public Color rysuj() {
        return Color.BLACK;
    }

    @Override
    public String getNazwa() {
        return "wilk";
    }
}
